### DISCUSSION TITLE
Consumption in drives

### DISCUSSION THREAD

**Discussion by binalbruno** [teslamate-org/teslamate#1187] [Q&A] [answered]
I am totally new to Teslamate. I was wondering why the consumption figures are empty for all my drives. Could any one help me?
![Screenshot 2021-01-08 145222](https://user-images.githubusercontent.com/77157746/104022842-1fdc4780-51c1-11eb-9685-6116b2f80367.png)

**Comment by adriankumpf** [comment, marked as answer]
This is perfectly normal. Here is the answer to this: [FAQ – Why are no consumption values displayed in Grafana?](https://docs.teslamate.org/docs/faq#why-are-no-consumption-values-displayed-in-grafana). In short, it takes at least two charging sessions before the first estimate can be displayed.

**Reply by binalbruno** [reply]
Super cool. 
BTW, thanks a lot for making this data logger. It is really awesome.
Till now I was manually entering the logs into a google sheet and using overcomplicated pivot tables. :|
